defog_data/supplementary.py

```diff
@@ -545,8 +545,86 @@ def load_embeddings(emb_path: str) -> tuple[dict, dict]:
+        ],
+        ("customers", "sales"): [
+            ("customers.id", "sales.customer_id")
+        ],
+        ("cars", "inventory_snapshots"): [
+            ("cars.id", "inventory_snapshots.car_id")
+        ],
+        ("sales", "payments_received"): [
+            ("sales.id", "payments_received.sale_id")
+        ]
+    },
+    "derm_treatment": {
+        ("patients", "treatments"): [
+            ("patients.patient_id", "treatments.patient_id")
+        ],
+        ("doctors", "treatments"): [
+            ("doctors.doc_id", "treatments.doc_id")
+        ],
+        ("drugs", "treatments"): [
+            ("drugs.drug_id", "treatments.drug_id")
+        ],
+        ("diagnoses", "treatments"): [
+            ("diagnoses.diag_id", "treatments.diag_id")
+        ],
+        ("treatments", "outcomes"): [
+            ("treatments.treatment_id", "outcomes.treatment_id")
+        ],
+        ("treatments", "adverse_events"): [
+            ("treatments.treatment_id", "adverse_events.treatment_id")
+        ],
+        ("treatments", "concomitant_meds"): [
+            ("treatments.treatment_id", "concomitant_meds.treatment_id")
+        ]
+    },
+    "ewallet": {
+        ("consumer_div.users", "consumer_div.notifications"): [
+            ("consumer_div.users.uid", "consumer_div.notifications.user_id")
+        ],
+        ("consumer_div.users", "consumer_div.user_sessions"): [
+            ("consumer_div.users.uid", "consumer_div.user_sessions.user_id")
+        ],
+        ("consumer_div.users", "consumer_div.user_setting_snapshot"): [
+            ("consumer_div.users.uid", "consumer_div.user_setting_snapshot.user_id")
+        ],
+        ("consumer_div.users", "consumer_div.wallet_user_balance_daily"): [
+            ("consumer_div.users.uid", "consumer_div.wallet_user_balance_daily.user_id")
+        ],
+        ("consumer_div.users", "consumer_div.wallet_transactions_daily"): [
+            ("consumer_div.users.uid", "consumer_div.wallet_transactions_daily.sender_id"),
+            ("consumer_div.users.uid", "consumer_div.wallet_transactions_daily.receiver_id")
+        ],
+        ("consumer_div.merchants", "consumer_div.wallet_transactions_daily"): [
+            ("consumer_div.merchants.mid", "consumer_div.wallet_transactions_daily.sender_id"),
+            ("consumer_div.merchants.mid", "consumer_div.wallet_transactions_daily.receiver_id")
+        ],
+        ("consumer_div.merchants", "consumer_div.coupons"): [
+            ("consumer_div.merchants.mid", "consumer_div.coupons.merchant_id")
+        ],
+        ("consumer_div.merchants", "consumer_div.wallet_merchant_balance_daily"): [
+            ("consumer_div.merchants.mid", "consumer_div.wallet_merchant_balance_daily.merchant_id")
+        ],
+        ("consumer_div.coupons", "consumer_div.wallet_transactions_daily"): [
+            ("consumer_div.coupons.cid", "consumer_div.wallet_transactions_daily.coupon_id")
+        ]
```

Comment:
2 more to add:
`consumer_div.coupons.merchant_id` can be joined with `consumer_div.wallet_transactions_daily.sender_id` and `consumer_div.wallet_transactions_daily.receiver_id`